client/ayon_maya/plugins/publish/collect_cache_farm.py

```diff
@@ -0,0 +1,19 @@
+import pyblish.api
+from ayon_maya.api import plugin
+
+
+class CollectDataforCache(plugin.MayaInstancePlugin):
+    """Collect data for caching to Deadline."""
```

Comment:
If this targets deadline - shouldn't this be in `ayon-deadline`?

Also, this data is SO generic? Why does it need a Collector of it's own.

- `instance.data["publish"] = True` seems like a given, and even what should be default behavior?
- `"plugin": "MayaBatch"` seems awfully Deadline specific and why can't `MayaBatch` be the default plug-in in `MayaCacheSubmitDeadline`?
With that we can remove this full plug-in, no?